doc/man1/dgst.pod

```diff
@@ -8,7 +8,7 @@ dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blak
 
 B<openssl> B<dgst>
 [B<-help>]
-[B<-I<digest>>]
+[B<-[digest]>]
```

Comment:
Commit https://github.com/openssl/openssl/pull/4144/commits/5fe7a8c6b930088ddff23e54283a9c9caa83dafa tries to address this.